internal/kubernetes/podfilters.go

```diff
@@ -79,6 +79,16 @@ func NewDaemonSetPodFilter(client kubernetes.Interface) PodFilterFunc {
 	}
 }
 
+// UnprotectedPodFilter returns a FilterFunc that returns true if the
+// supplied pod does not have the user-specified annotation for protection from
+// eviction
+func UnprotectedPodFilter(annotationKey string) PodFilterFunc {
```

Comment:
I briefly considered the idea, but could not figure out the best way to setup the CLI. In the end, I ended up going with something simpler. Perhaps something like `--protected-pod-annotation=annotation-key=annotation-value` could work?

Reply:
Yeah, that's what I was thinking. You can use the `--node-label` flag as a reference for that. Two things to watch out for when using a `StringMap()` flag:

* It's best to use a singular flag name (i.e. `protected-pod-annotation` not `protected-pod-annotations`), because the CLI UX is to repeat the flag multiple times.
* Kingpin doesn't give a very useful example by default so I like to explicitly set a placeholder of `KEY=VALUE`.

Reply:
Could use a little input on this. I ended up going with `Strings()` instead of `StringMap()` for the extra functionality of defining annotations with optional values: `KEY[=VALUE]`. However, I realize this is different from the `--node-label` argument. Do you think `StringMap()` would be better here? I'm happy to change it.